src/activity_handlers.cpp

```diff
@@ -2121,6 +2123,20 @@ void activity_handlers::chop_tree_finish( player_activity *act, player *p ) {
     act->set_to_null();
 }
 
+void activity_handlers::chop_logs_finish( player_activity *act, player *p ) {
+    const tripoint &pos = act->placement;
+
+    g->m.ter_set( pos, t_dirt );
```

Comment:
Not necessary hear, but just realized it'd be neat to have a stump item here, might even be useful, it could have the flag for resting a longarm on.